documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -779,9 +779,7 @@ try {
 The correct way to implement this is to use
 `cache.getAdvancedCache().withFlags(Flag.FORCE_WRITE_LOCK).get(k)`.
 
-In caches with optimistic locking writes can return a stale previous value as well, and
-the only way protect against it is to enable write-skew checks and to catch
-`WriteSkewException`.
+In caches with optimistic locking, writes can also return stale previous values. The only way to avoid stale previous values is to enable write-skew checks to catch `WriteSkewException`.
```

Comment:
This sounds like you must enable write skew checks explicitly, but that's no longer true (since version 8.x or so). Write skew checks are on by default; you can disable them as a performance optimization.